README.md

```diff
@@ -7,19 +7,21 @@ Yet another F-Droid client.
 
 ## Description
 
-Unofficial F-Droid client that resembles the classic F-Droid client.
+Unofficial F-Droid client resembling the classic F-Droid client.
```

Comment:
```suggestion
Unofficial F-Droid client in the style of the classic one.
```
Could "Unofficial" be replaced with "Alternative". The F-Droid folks are all about alternative clients.